### PR TITLE
Bridge relayer client

### PR DESCRIPTION
**Pull request by fewensa**
Related: https://github.com/darwinia-network/docs/issues/231
> Relayer clients/Bridge Instances (bridge-relayer-client.md)

### CODE REVIEW

**Review by 1022ranji — APPROVED** [review]

File: docs/bridge-knowledge-base/bridge-relayer-client.md [hunk 1]
```diff
@@ -5,4 +5,34 @@ sidebar_label: Relayer Clients
...
+Currently, About Darwinia we have the following related bridges.
+
+| Bridge             | Description                             |
+| ------------------ | --------------------------------------- |
+| Darwinia &lt;&gt; Ethereum | Implemented and running                 |
+| Darwinia &lt;&gt; Crab     | Planning, We already have a test bridge |
+| Darwinia &lt;&gt; Bsc      | Planning                                |
+
+## Bridger
+All bridges are implemented in the bridger project. The bridger project has been designed as a common bridge solution. The bridger also supports multiple bridges running at the same time and have convenient event scheduling system.
```
Comment (robinwei-itering) [inline docs/bridge-knowledge-base/bridge-relayer-client.md]: 1. common -> generic
2. ...and have convenient... -> ...and has convenient...

File: docs/bridge-knowledge-base/bridge-relayer-client.md [hunk 2]
```diff
@@ -5,4 +5,34 @@ sidebar_label: Relayer Clients
...
+
+For more information about bridger, you can see
+
+- Code: [darwinia-network/bridger](https://github.com/darwinia-network/bridger/)
+- Document: [Bridger Guide](https://itering.notion.site/Bridger-Guide-en-b5ae438609df41f682ab0efce3ddb98b)
+
+### Bridge Darwinia &lt;&gt; Ethereum
+
+The Darwinia has implemented Ethereum bridge and is already in work. This is a two-way bridge that can freely relay transactions between the two parties to the other chain.
```
Comment (robinwei-itering) [inline docs/bridge-knowledge-base/bridge-relayer-client.md]: Darwinia has implemented the Ethereum bridge, and it is already in operation. It is a two-way bridge that can relay transactions between the two networks back and forth.

File: docs/bridge-knowledge-base/bridge-relayer-client.md [hunk 3]
```diff
@@ -5,4 +5,34 @@ sidebar_label: Relayer Clients
...
+
+- Code: [darwinia-network/bridger/task/task-darwinia-ethereum](https://github.com/darwinia-network/bridger/tree/master/task/task-darwinia-ethereum)
+
+### Bridge Darwinia &lt;&gt; Crab
+
+This is the bridge we are planning. The test bridge [Pangolin &lt;&gt; Pangoro](https://github.com/darwinia-network/bridger/blob/master/task/task-pangolin-pangoro) has been completed.
+After the test bridge is stable, it will be implemented between Darwinia and Crab.
```
Comment (robinwei-itering) [inline docs/bridge-knowledge-base/bridge-relayer-client.md]: After the test bridge is stable -> When the test bridge becomes stable

File: docs/bridge-knowledge-base/bridge-relayer-client.md [hunk 4]
```diff
@@ -5,4 +5,34 @@ sidebar_label: Relayer Clients
...
+In the test bridge, we implemented an on-demand relay and implemented an incentive strategy.
```
Comment (robinwei-itering) [inline docs/bridge-knowledge-base/bridge-relayer-client.md]: In the test bridge, we have implemented the on-demand relay mechanism and an incentivisation strategy.

**Review by robinwei-itering — CHANGES_REQUESTED** [review]

**Pull request merged**